### PR TITLE
Added libtremor.

### PR DESCRIPTION
**Pull request by Rinnegatamante**

### CODE REVIEW

**Comment by xyzz** [comment]
What's the use of this library?

**Comment by Rinnegatamante** [comment]
Ogg vorbis decoding.

**Comment by xyzz** [comment]
Why use it over libvorbis?

**Comment by Rinnegatamante** [comment]
Faster and lower memory usage. This is the recommended lib for embedded system by xiph foundation.

File: libtremor/VITABUILD [hunk 1]
```diff
@@ -0,0 +1,18 @@
+pkgname=libtremor
+pkgver=9999
+pkgrel=1
+url="http://xiph.org/"
+source=("http://downloads.sourceforge.net/sourceforge/devkitpro/tremor-lowmem-src.tar.bz2")
```
Comment (devnoname120) [inline libtremor/VITABUILD]: Please use a tarball of a specific version instead.

File: libtremor/VITABUILD [hunk 2]
```diff
@@ -0,0 +1,18 @@
+pkgname=libtremor
+pkgver=9999
```
Comment (devnoname120) [inline libtremor/VITABUILD]: And fix the version.

**Pull request merged**